R/bootstrap.nlsfit.R

```diff
@@ -182,6 +182,7 @@ get.errors <- function (useCov, y, dy, dx, CovMatrix, errormodel, bsamples, cov_
     }
 
     if (is.null(CovMatrix)) {
+      CovMatrix <- cov_fn(bsamples)
       InvCovMatrix <- try(invertCovMatrix(bsamples, boot.l = 1, boot.samples = TRUE, cov_fn = cov_fn), silent = TRUE)
```

Comment:
```suggestion
      InvCovMatrix <- try(solve(CovMatrix), silent = TRUE)
```
Or even better calculate the covariance matrix, if missing. Then invert and cholesky decompose. No redundancies in the `if...else`.

Reply:
The reason that I didn't unify the two logic branches was the following: I assume that when one bothers to pass a hand-crafted covariance matrix, one might have made sure that it can be inverted naively. On the other hand, when the covariance matrix is computed simply from the bootstrap samples, one might want to use the SV decomposition with small eigenvalue replacement. What do you think?

Reply:
I've added some comments and extended the documentation somewhat to clarify things.

Reply:
I see. It was not clear to me that the algorithms differ that significantly.